### PR TITLE
Kendal_spark_implementation

### PR DESCRIPTION
**Pull request by rishabsinghh**
I have implemented Kendal's Correlation using spark. There are still some optimizations to be made. But for now this works.

### CODE REVIEW

**Review by fabclmnt — CHANGES_REQUESTED** [review]
@rishabsinghh please create the PR to the `spark-branck`. Currently, no spark development is yet available in develop and master branches. 

Also, can you please update you PR commit message to follow the expected commit format, for example:
'feat: (description)'

File: src/pandas_profiling/model/pandas/correlations_pandas.py [hunk 1]
```diff
@@ -6,6 +6,11 @@
 import numpy as np
 import pandas as pd
 from scipy import stats
+from operator import add 
+import pyspark
+from pyspark import SparkContext, SparkConf
+import numpy as np 
+from collections import defaultdict
```
Comment (sbrugman) [inline src/pandas_profiling/model/pandas/correlations_pandas.py:13]: Please run the pre-commit hooks

File: src/pandas_profiling/model/pandas/correlations_pandas.py [hunk 2]
```diff
@@ -35,7 +40,68 @@ def pandas_pearson_compute(
 def pandas_kendall_compute(
     config: Settings, df: pd.DataFrame, summary: dict
 ) -> Optional[pd.DataFrame]:
-    return df.corr(method="kendall")
+
```
Comment (sbrugman) [inline src/pandas_profiling/model/pandas/correlations_pandas.py:43]: The spark implementation should be placed in `src/pandas_profiling/model/spark/correlation_spark.py`

**Comment by chanedwin** [comment]
@rishabsinghh please change the PR's branch to feat/spark-minimal, thanks###

**Comment by chanedwin** [comment]
moved over to https://github.com/ydataai/pandas-profiling/pull/1205

**Pull request closed without merging**